simulation/traffic_simulator/include/traffic_simulator/math/hermite_curve.hpp

```diff
@@ -66,7 +66,8 @@ class HermiteCurve
     geometry_msgs::msg::Point point0, geometry_msgs::msg::Point point1,
     bool search_backward = false) const;
   boost::optional<double> getCollisionPointIn2D(
-    std::vector<geometry_msgs::msg::Point> polygon, bool search_backward = false) const;
+    std::vector<geometry_msgs::msg::Point> polygon, bool search_backward = false,
+    bool close_start_end = true) const;
```

Comment:
`std::vector<geometry_msgs::msg::Point>` => `const std::vector<geometry_msgs::msg::Point> &`

Reply:
fixed in 7da4bc96c3a07a367967a2f1f797974c3855c5d7